contracts/Governor/GovernorFactory.sol

```diff
@@ -64,14 +64,13 @@ contract GovernorFactory is ERC165, ModuleFactoryBase {
                 address(abi.decode(data[3], (address))),
                 abi.encodeWithSelector(
                     IGovernorModule(payable(address(0))).initialize.selector,
-                    abi.decode(data[5], (string)),
                     abi.decode(data[2], (address)),
                     timelock,
-                    abi.decode(data[6], (uint64)),
+                    abi.decode(data[5], (uint64)),
+                    abi.decode(data[6], (uint256)),
                     abi.decode(data[7], (uint256)),
                     abi.decode(data[8], (uint256)),
                     abi.decode(data[9], (uint256)),
-                    abi.decode(data[10], (uint256)),
                     abi.decode(data[1], (address))
```

Comment:
Do you mind commenting on what these changes are?

Reply:
Yes, so the name was being passed in the data[5] index. That got removed, so all of the array indexes above 5 needed to be shifted down to fill in that empty array index.